src/main/java/core/basesyntax/strategy/impl/PurchaseOperationImpl.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.FruitTransaction;
+import core.basesyntax.database.StorageOfFruits;
+import core.basesyntax.strategy.OperationHandler;
+
+public class PurchaseOperationImpl implements OperationHandler {
+    @Override
+    public void operation(FruitTransaction fruitTransaction) {
+        int countOfFruit = StorageOfFruits.get(fruitTransaction.getFruit());
```

Comment:
What happen if fruitTransaction.getQuantity() is more than you have fruits in the storage?